packages/react-core/src/Authenticator/hooks/types.ts

```diff
@@ -75,6 +75,7 @@ export type CommonRouteProps = {
   handleBlur: UseAuthenticator['updateBlur'];
   handleChange: UseAuthenticator['updateForm'];
   handleSubmit: UseAuthenticator['submitForm'];
+  validationErrors?: UseAuthenticator['validationErrors'];
```

Comment:
Not sure if this the correct path forward. Why not use the `ValidationProps` interface?

Reply:
Wasn't available when I was working on it :)  Will adjust.